packages/learner-access/src/components/LoadUserLearning.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { useCallback, useState } from 'react';
+
+import { LoadedComponentProps } from '../types';
+import {
+  RightArrowIcon,
+  DownArrowIcon,
+  HelpIcon,
+  StopwatchIcon,
+  ChatIcon,
+  ViewIcon,
+  FileIcon
+} from '../Assets/Icons';
+import {
+  useUserContentItemsQuery,
+  useUserCourseProgressQuery,
+  useUserCourseCompletionProgressQuery,
+  useUserCourseCollaborationsQuery,
+  useUserCourseAwardCountsQuery,
+  LoadingDots,
+  GlobalTypes,
+  hydrateContent,
+  HydratedContentItem,
+  formatTime
+} from '@thoughtindustries/content';
+import { ArchiveButton } from './MutationCallingButtons';
+import { Tooltip } from '../Assets/Tooltips';
+import { useTranslation } from 'react-i18next';
+import useLearnerAccess from '../use-context';
+import { t } from 'i18next';
```

Comment:
Don't need to import `t` here since we're using the `useTranslation` hook on line 42. You can just update line 42 to 


```javascript
const { i18n, t } = useTranslation();
```